.dockstore.yml

```diff
@@ -173,7 +173,7 @@ workflows:
        branches:
```

Comment:
All the changes in this file were just for testing purposes and don't need to be merged.